compiler-rt/lib/interception/interception_linux.cpp

```diff
@@ -11,6 +11,8 @@
 // Linux-specific interception methods.
 //===----------------------------------------------------------------------===//
 
+#define SANITIZER_COMMON_NO_REDEFINE_BUILTINS
```

Comment:
this one is weird.
if we compile this into full sanitizers, we want redefine
if it's part of safe stack we want no-redefine

what if keep interception as is, but in safe stack use both:

1. #define SANITIZER_COMMON_NO_REDEFINE_BUILTINS

2. define *__sanitizer_internal_mem* forwarded to libc, to cover interception

```
extern "C" {
SANITIZER_INTERFACE_ATTRIBUTE void *__sanitizer_internal_memcpy(void *dest,
                                                                const void *src,
                                                                uptr n) {
  return memcpy();
}

SANITIZER_INTERFACE_ATTRIBUTE void *__sanitizer_internal_memmove(void *dest,
                                                                const void *src,
                                                                uptr n) {
  return memmove();
}

SANITIZER_INTERFACE_ATTRIBUTE void *__sanitizer_internal_memmove(void *dest,
                                                                const void *src,
                                                                uptr n) {
  return memset();
}
```

Reply:
This works just fine, re-tested as before.